.github/workflows/main.yaml

```diff
@@ -0,0 +1,30 @@
+name: main
+
+on:
+  push:
+    branches: [ 'develop','staging','prod' ]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-22.04
+    # context environment variables are setup in github to match branch names
+    environment:
+      name: ${{ github.ref_name }}
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: BSFishy/pip-action@v1
+        with:
+          packages: |
+            synapseclient[pandas,pysftp]
+      - name: Setup mongo tools - mongo-tools and mongo-shell
+        uses: boly38/action-mongo-tools@stable
+      - name: Import Synapse Data
+        run: ./import-data.sh $BRANCH $SYNAPSE_USERNAME $SYNAPSE_PASSWORD $DB_HOST $DB_USER $DB_PASS
+        env:
+          BRANCH: ${{ github.ref_name }}
+          SYNAPSE_USERNAME: ${{ secrets.SYNAPSE_USERNAME }}
+          SYNAPSE_PASSWORD: ${{ secrets.SYNAPSE_PASSWORD }}
+          DB_HOST: ${{ vars.DB_HOST }}
```

Comment:
Secrets?

Reply:
> Aren't these env secrets?

yes, take a look at the screenshot in set of files for this PR.

Reply:
Sorry my point was should they not be referred to as ${{ secrets.DB_HOST }}?

Reply:
nope. the doc says to use `vars` when used in an environment context.  i have tested it and `vars` seems to work.

Reply:
ohh, oops.  i was testing with env vars.  sorry you are correct.  i will need to fix this

Reply:
ok, fixed.